src/d_main.cpp

```diff
@@ -567,6 +567,23 @@ CVAR (Flag, sv_alwaysspawnmulti,	dmflags2, DF2_ALWAYS_SPAWN_MULTI);
 CVAR (Flag, sv_novertspread,		dmflags2, DF2_NOVERTSPREAD);
 CVAR (Flag, sv_noextraammo,			dmflags2, DF2_NO_EXTRA_AMMO);
 
+//==========================================================================
+//
+// CVAR dmflags3
+//
+//==========================================================================
+
+CUSTOM_CVAR(Int, dmflags3, 0, CVAR_SERVERINFO | CVAR_NOINITCALL)
+{
+	if ((self & DF3_NO_PLAYER_CLIP) && (!multiplayer || deathmatch))
+	{
+		self = self & ~DF3_NO_PLAYER_CLIP;
+		Printf("No player clipping is only available in coop\n");
```

Comment:
I don't really see any real benefit for this blocker, and it also is inconsistent with how other dmflags work, would prefer not to have this restriction.

Reply:
@madame-rachelle I agree about it being different than the other 2 dmflags and, true, this code here is unnecessary as the code everywhere else is already doing `(multiplayer && !deathmatch)`, therefore setting the CVar will do nothing, [i]anyway[/i]. However I thought it would be user-friendly to tell the user that this flag is strictly designed for coop (makes zero sense to disable player-blocking in deathmatch).

If you feel too strongly against this, I can remove this specific block.